core/scripts/cre/environment/environment/workflow.go

```diff
@@ -32,8 +32,23 @@ const (
 	DefaultWorkflowOwnerAddress = "0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266"
 )
 
-var defaultWorkflowRegistryTypeVersion = deployment.TypeAndVersion{
-	Version: *semver.MustParse("1.0.0"),
+// getWorkflowRegistryTypeVersion returns the appropriate TypeAndVersion based on the contracts version flag
+func getWorkflowRegistryTypeVersion(contractsVersion string) deployment.TypeAndVersion {
+	switch strings.ToLower(contractsVersion) {
+	case "v1":
+		return deployment.TypeAndVersion{
+			Version: *semver.MustParse("1.0.0"),
+		}
+	case "v2":
+		return deployment.TypeAndVersion{
+			Version: *semver.MustParse("2.0.0"),
```

Comment:
actually needs to be "2.0.0-dev", there is a constant in one of the packages.  Please reuse that.